talkative-tangs/Bangazon-LLC-2
# User Order History/Detail

Closes #17

Description of Proposed Changes
Adds option to "My Account" to view order history. Then, user can click order number for order details including products and total cost of order.

Steps to Test
git fetch --all
git checkout ed-order-history

Navigate to My Account and click on each available order. Some users may not have orders and pages should render without content.

Impacted Areas in Application
My Account
urls
views
new templates
readded product type selection to sell a product that was lost in a previous merge
added humanize to installed apps for price formatting (I used this for the total on order detail)

To review:
@ousamasama 
@laboyd001 
@BryanNilsen
